routes/favicon.coffee

```diff
@@ -0,0 +1,27 @@
+request = require 'request'
+fs = require 'fs'
+tmpdir = require 'os'
+	.tmpdir()
```

Comment:
Oh, and personally if I *did* want to write some code like what you have here I'd do it on one line, like `tmpdir = require('os').tmpdir()`.